uvloop/loop.pyx

```diff
@@ -3116,6 +3116,22 @@ cdef class Loop:
         await waiter
         return udp, protocol
 
+    def monitor_fs(self, path: str, callback, flags: int) -> asyncio.Handle:
+        p_bytes = path.encode('UTF-8')
+        cdef char* c_str_path = p_bytes
+        return self._monitor_fs(c_str_path, flags, callback)
+
+    cdef _monitor_fs(self, char* path, int flags, callback):
+        cdef:
+            UVFSEvent fsevent
+
+        self._check_closed()
+
+        return UVFSEvent.new(self, path, callback, flags)
+
+    def get_uvloop_ptr(self) -> int:
```

Comment:
Unused? Also if this was to be used in other Cython modules, consider using `cimport` to [link across them](https://cython.readthedocs.io/en/stable/src/userguide/sharing_declarations.html).

Reply:
so I didn't call this out but I would indeed like to use it in other modules, including ones not necessarily written as Cython

Reply:
> other modules, including ones not necessarily written as Cython

Hmm, I'm a bit unsure about introducing C pointers into Python as `int`, considering the "other modules" would more-or-less have C code to link to libuv, right?

I think I'm more comfortable with an external C-API like this (in `loop.pyx`):

```cython
cdef extern uv.uv_loop_t* get_uv_loop_ptr(PyObject* loop) nogil:
    return (<Loop>loop).uvloop
```

Then at the caller, you could use this C-API (linking to the `uvloop/loop.*.so`):

```c
uv_loop_t *get_uv_loop_ptr(PyObject *__pyx_v_loop);
```

Reply:
ok, done as you indicated, updated unit test, which let's <hope> will work